Welp it makes the linter unhappy :(
```
/home/runner/work/godot/godot/platform/javascript/js/libs/library_godot_audio.js
  232:7  error  Missing trailing comma  comma-dangle
```

--- a/platform/javascript/.eslintrc.js
+++ b/platform/javascript/.eslintrc.js
@@ -39,5 +39,13 @@ module.exports = {
 		// Closure compiler (exported properties)
 		"quote-props": ["error", "consistent"],
 		"dot-notation": "off",
+		// No comma dangle for functions (it's madness, and ES2017)
+		"comma-dangle": ["error", {
+			"arrays": "always-multiline",
+			"objects": "always-multiline",
+			"imports": "always-multiline",
+			"exports": "always-multiline",
+			"functions": "never"
+		}],
 	}
 };

--- a/platform/javascript/js/libs/library_godot_audio.js
+++ b/platform/javascript/js/libs/library_godot_audio.js
@@ -229,7 +229,7 @@ const GodotAudioWorklet = {
 					'godot-processor',
 					{
 						'outputChannelCount': [channels],
-					},
+					}
 				);
 				return Promise.resolve();
 			});

